opbeans-dotnet/Controllers/RootController.cs

```diff
@@ -0,0 +1,16 @@
+using System.Linq;
```

Comment:
Just a minor thing to follow our .NET code style: you can remove all the `using`s except `using Microsoft.AspNetCore.Mvc;`.